ui/components/app/permission-cell/permission-cell.js

```diff
@@ -107,7 +118,12 @@ const PermissionCell = ({
             'permission-cell__title-revoked': revoked,
           })}
         >
-          {title}
+          {invoker ? (
+            <Box display={Display.InlineBlock} size={Size.MD} marginRight={1}>
```

Comment:
Couldn't figure out how to set a different variant inside of the <Text> component.

Reply:
Why are we updating this file?